### ISSUE TITLE
Add dbb.file.tagging 

### ISSUE THREAD

**Issue by dennis-behm** [IBM/dbb-zappbuild#7]
With DBB 1.0.8, the CopyToHFS command was extended enable automated file tagging of NON-ASCII files. This leverages a new environment variable `dbb.file.tagging`. 

I raise this issue to introduce the environment variables in the dbb-zappbuild sample.

For more details, please have a look at:
https://www.ibm.com/support/knowledgecenter/SS6T76_1.0.8/scriptorg.html

**Comment by drbruce-git** [comment]
This is the text added to the old MortgageApplication build.properties
```
# dbb.file.tagging controls compile log and build report file tagging. If true, files 
# written as UTF-8 or ASCII are tagged. The default value is false. If the environment
# variable _BPXK_AUTOCVT is set ALL, file tagging may have an adverse effect if viewing
# log files and build report via Jenkins. In this case, set dbb.file.tagging to false or
# comment out the line.
dbb.file.tagging=true
```

**Comment by dennis-behm** [comment]
Resolved via #33
